stl/inc/chrono

```diff
@@ -5136,54 +5136,95 @@ namespace chrono {
         return _Istr;
     }
 
+    template <class _CharT, class _Traits, class _Alloc, class _Parsable>
+    struct _Time_parse_iomanip_c_str {
+        _Time_parse_iomanip_c_str(const _CharT* _Fmt_, _Parsable& _Tp_,
+            basic_string<_CharT, _Traits, _Alloc>* _Abbrev_ = nullptr, minutes* _Offset_ = nullptr)
+            : _Fmt{_Fmt_}, _Tp{_Tp_}, _Abbrev{_Abbrev_}, _Offset{_Offset_} {}
+        _Time_parse_iomanip_c_str(_Time_parse_iomanip_c_str&&) = delete;
+
+        const _CharT* _Fmt;
+        _Parsable& _Tp;
+        basic_string<_CharT, _Traits, _Alloc>* _Abbrev;
+        minutes* _Offset;
+    };
+
     template <class _CharT, class _Traits, class _Alloc, class _Parsable>
     struct _Time_parse_iomanip {

```

Comment:
Maybe a `// TRANSITION, ABI` comment to the effect that this can be replaced with `_Time_parse_iomanip_c_str` in the future?

Reply:
The small code duplication is not bad enough to worry me, personally. Storing a reference to `basic_string` also has a tiny benefit, that is the format string is not invalidated if the `basic_string` is resized after the call to `parse`.